test/legacy_test/test_nonzero_api.py

```diff
@@ -181,5 +197,37 @@ def return_outputs(self):
         return {'Out': np.transpose(np.nonzero(self.inputs['Condition']))}
 
 
+class TestZeroOp(OpTest):
```

Comment:
```suggestion
class TestNonzeroZeroSizeOp(OpTest):
```
此处应该是 nonzero api 的 0-size 测试……而不是 zero

Reply:
哦哦 写漏了